snowshu/adapters/target_adapters/postgres_adapter/functions/UUID_STRING.sql

```diff
@@ -1,4 +1,6 @@
 /* emulates snowflake's uuid_string() function for v4 UUIDs, v5 unsupported */
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
Worth maybe at least noting (if not implementing at this time):
bigquery needs `GENERATE_UUID()`
https://cloud.google.com/bigquery/docs/reference/standard-sql/uuid_functions
and then this works for everyone that supports it (there's not a native UUID in RS).

Reply:
Currently, snowshu doesn't support bigquery at all as far as I can tell. It looks like you've got bigquery work in branches of your fork, but I don't think any of it was ever merged into upstream.